main.cpp

```diff
@@ -4594,6 +4594,29 @@ void sign_guts_elf(elf_file* elf, private_t private_key, public_t public_key) {
         new_block.items.push_back(version);
     }
 
+    // Add entry point when signing Arm images
+    std::shared_ptr<image_type_item> image_type = new_block.get_item<image_type_item>();
+    if (settings.seal.sign && image_type != nullptr && image_type->image_type() == type_exe && image_type->cpu() == cpu_arm) {
+        std::shared_ptr<entry_point_item> entry_point = new_block.get_item<entry_point_item>();
+        if (entry_point == nullptr) {
+            std::shared_ptr<vector_table_item> vtor = new_block.get_item<vector_table_item>();
+            uint32_t vtor_loc = 0x10000000;
```

Comment:
should come from wherever the start of the image is; which in flash binaries is affected by roll; and would be in RAM for RAM binaries (where i guess it must be at 0x20000000 in these cases)

Reply:
Fixed - I've added support for adding the rolling window delta to the VTOR location, and it checks if the ELF entry point is in SRAM and reads the VTOR from there if that's the case

Reply:
cool; note i looked at the code, but didn't check that the roll was in the right direction ;-)